### PR TITLE
Make height and width respect all the interface

### PR DESCRIPTION
**Pull request by davesnx**
- Fixed fit-content
- Added maxWidth/minWidth support for MaxWidth and MinWidth constructors
- Added maxHeight/minHeight for MaxHeight and MinHeight constructors
- Fixed build for dev-server-dom/dev-server-fela/dev-server-emotion since they were using the CssJs and lacked uncurried notation

### CODE REVIEW

File: bs-css/src/Css_AtomicTypes.res [hunk 1]
```diff
@@ -1615,18 +1615,18 @@ module TextDecorationThickness = {
 }
 
 module Width = {
-  type t = [#auto | #fitContent | #maxContent | #minContent]
+  type t = [#auto | #fitContent(Length.t) | #maxContent | #minContent]
```
Comment (davesnx) [inline bs-css/src/Css_AtomicTypes.res:1618]: This is technically a breaking change. Not entirely sure if it's used anywhere since it was added recently

**Comment by davesnx** [comment]
I would recommend to add a CI step to build all folders since it looks like some diffs aren't running the build and checking it. It can cause a breakage if it publishes at that time.

**Comment by davesnx** [comment]
Would be nice to have this released with https://github.com/giraud/bs-css/pull/257. WDYT @giraud?

**Comment by giraud** [comment]
Yes, will do next week

**Pull request merged**